metadata-service/war/src/main/resources/boot/policies.json

```diff
@@ -45,7 +45,8 @@
       },
       "privileges":[
         "EDIT_ENTITY",
-        "VIEW_ENTITY_PAGE"
+        "VIEW_ENTITY_PAGE",
+        "EDIT_LINEAGE"
```

Comment:
technically this isn't necessary since if you have `EDIT_ENTITY` you can also edit the lineage

Reply:
i guess that's the case for all of these...

but I'm not against being explicit!